docs/contributor/resources/03-moduletemplate.md

```diff
@@ -1,6 +1,14 @@
-# ModuleTemplate Custom Resource
+# ModuleTemplate
 
-The core of our modular discovery is the [ModuleTemplate custom resource (CR)](../../../api/v1beta2/moduletemplate_types.go). It is used to initialize and resolve modules.
+The `moduletemplates.operator.kyma-project.io` Custom Resource Definition (CRD) is a comprehensive specification that defines the structure and format used to configure the ModuleTemplate resource.
+
+The Manifest CR contains modules' metadata with links to their images and manifests. ModuleTemplate CR represents a module in a particular version. Based on this resource Lifecycle Manager enables or disables modules in a cluster. It contains the list of added modules and their state.
```

Comment:
I think a few things got tangled up here.

> The Manifest CR contains modules' metadata with links to their images and manifests. ModuleTemplate CR represents a module in a particular version.

Are you sure you want to say `Manifest CR`? And overall this section will only be true fully once we released the new ModuleTemplate CR changes. But for me it would be okay to add the description already as proposed.

> Based on this resource Lifecycle Manager enables or disables modules in a cluster. It contains the list of added modules and their state.

I think this rather describes the module list in spec and status of KymaCR. Here, I would rather expect something like *ModuleTemplates define the set of module that can be installed by Lifecycle Manager*.

> Manifest CR [...] ModuleTemplate CR [...]

in manifest.md and watcher.md you also introduced the abbreviation: `[...] custom resource (CR)`

Reply:
As we agreed, let's rephrase to:

> The ModuleTemplate custom resource (CR) defines a module, in a particular version, that can be added to or deleted from the module list in the Kyma CR. Each ModuleTemplate CR represents one module.